src/components/chip/examples/chip-button.tsx

```diff
@@ -0,0 +1,72 @@
+import { Component, State, h } from '@stencil/core';
+
+/**
+ * Chip as button
+ * Typically, a chip is used to trigger an action or act as an input element.
+ * This is why the component generates a `<button>` element in the DOM.
+ * You will only need to handle the `onClick` event on the component.
+ */
+@Component({
+    tag: 'limel-example-chip-button',
+    shadow: true,
+})
+export class ChipButtonExample {
+    @State()
+    private disabled: boolean = false;
+
+    @State()
+    private selected: boolean = false;
+
+    @State()
+    private readonly: boolean = false;
+
+    public render() {
+        return [
+            <limel-chip
+                id="1"
+                text="Click me!"
+                icon="star"
+                onClick={this.onClick}
+                disabled={this.disabled}
+                selected={this.selected}
+                readonly={this.readonly}
+            />,
+            <limel-example-controls>
+                <limel-checkbox
+                    checked={this.disabled}
+                    label="Disabled"
+                    onChange={this.setDisabled}
+                />
+                <limel-checkbox
+                    checked={this.readonly}
+                    label="Readonly"
+                    onChange={this.setReadonly}
+                />
+                <limel-checkbox
+                    checked={this.selected}
+                    label="Selected"
+                    onChange={this.setSelected}
+                />
+            </limel-example-controls>,
+        ];
+    }
+
+    private onClick() {
+        console.log('Chip is clicked.');
```

Comment:
The current behaviour makes the chip in this example look _very_ much like a button, that you might want to use for triggering actions, just like we tell people they should _not_. I would suggest that we add a flip of the `selected` state here as well, to avoid that.

```suggestion
        console.log('Chip is clicked.');
        this.selected = !this.selected;
```

Reply:
But that's not what I want here. Toggling `selected` state is showcased in another example. 

Here, you should imagine a chip in a chip-set. You just wanna click it and something should happen. That's a hard fact and we can't change it. 

When I said I don't want people to use chips instead of buttons, I meant I don't want them for instance to use two chips like `❌ Cancel` and `💾 Save`, and put them in a modal. But I can't stop them from doing so neither. I think reading the docs should be enough. Maybe we can add a couple of DO and DON'T examples too.

In this example, maybe we can change the icon and label of the chip.

Reply:
OK. No worries, as long as you are happy with what's being conveyed, I'm perfectly OK with that 😊